sm/enclave.c

```diff
@@ -131,25 +131,14 @@ void enclave_init_metadata(){
   }
 }
 
-static enclave_ret_t init_enclave_memory(uintptr_t base, uintptr_t size,
-    uintptr_t utbase, uintptr_t utsize)
+static enclave_ret_t init_enclave_memory(uintptr_t utbase, uintptr_t utsize)
```

Comment:
Can you change the name of this function to something like `clean_initial_memory`..? (probably there's a better name..)
and also add memsetting freemem to 0?
I forgot to add zeroing out of the freemem

Reply:
Will do on the rename.
I'm not going to zero freemem (which VERY large in some cases) right now. See https://github.com/keystone-enclave/riscv-pk/issues/38